concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -1338,6 +1338,116 @@ jobs:
                   ./paas-cf/concourse/scripts/set_quotas_from_manifest.rb cf-manifest/cf-manifest.yml
 
       - aggregate:
+        - do:
+          - task: deploy-test-application
+            config:
+              platform: linux
+              image_resource:
+                type: docker-image
+                source:
+                  repository: governmentpaas/cf-cli
+              params:
+                DISABLE_HEALTHCHECK_DB: {{disable_healthcheck_db}}
+              inputs:
+                - name: paas-cf
+                - name: config
+                - name: bosh-CA
+              run:
+                path: sh
+                args:
+                  - -e
+                  - -u
+                  - -c
+                  - |
+                    ./paas-cf/concourse/scripts/import_bosh_ca.sh
+
+                    . ./config/config.sh
+                    echo | cf login -a ${API_ENDPOINT} -u ${CF_ADMIN} -p ${CF_PASS}
+                    cf create-org testers
+                    cf set-quota testers test_apps
+                    cf create-space test-application -o testers
+                    cf target -o testers -s test-application
+
+                    cd paas-cf/platform-tests/example-apps/healthcheck
+
+                    if  [ "${DISABLE_HEALTHCHECK_DB:-}" != "true" ] ; then
```

Comment:
I thought that was the case. But then I was considering, if we'd like to put more load on. Thanks. Removing.